hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/HoodieTable.java

```diff
@@ -153,6 +156,7 @@ protected HoodieTable(HoodieWriteConfig config, HoodieEngineContext context, Hoo
     this.index = getIndex(config, context);
     this.storageLayout = getStorageLayout(config);
     this.taskContextSupplier = context.getTaskContextSupplier();
+    this.txnManager = new TransactionManager(config, metaClient.getStorage());
```

Comment:
Not sure we introduce txnManager into hoodie table? It looks like it is only used for rollback, can we move it out?